src/compiler/target/golang/mod.rs

```diff
@@ -0,0 +1,268 @@
+use super::*;
+
+pub struct Go {}
+
+impl Go {
+  /// Generate golang code using tokens from parser
+  pub fn generate(lb: &mut LangBuilder, t: AnilizedTokens) -> Result<(), LocationError> {
+
+    // TODO: Replace when file importing is implemented
+    lb.code("package main");
+
+    let mut code = Self {};
+
+    // define functions
+    for (_, func) in t.functions {
+      code.function(func, lb);
+    }
+
+    // define types
+    for (_, type_) in t.types {
+      code.code_type(type_, lb);
+    }
+
+    // define structs
+    for (_, structure) in t.structs {
+      code.structure(structure, lb);
+    }
+    
+    // define globals
+    for (_, glob) in t.vars {
+      code.global_var(glob, lb);
+    }
+
+    Ok(())
+  }
+  /// Get the type as a string
+  pub fn get_type(&mut self, type_: Option<tokenize::types::Type>) -> String {
+    match type_ {
+      Some(res) => {
+        match res.type_ {
+          tokenize::types::TypeType::Char => "rune".to_string(),
```

Comment:
Firstly we don't need the `::types` here because `tokenize` exposes `TypeType` already :)  
Secondly we need to check this is almost all languages so we can better add this to the: `pub use tokenize` in `target/mod.go`  

Once that's dune we only have to have `TypeType::` here